ui/app/templates/components/secret-edit-display.hbs

```diff
@@ -36,17 +36,22 @@
 
   {{#if @showAdvancedMode}}
     <div class="form-section">
-      <label class="title is-5">
-        {{#if isV2}}
-        Version data
-        {{else}}
-        Secret data
-        {{/if}}
-      </label>
+      <Toolbar>
```

Comment:
I did that but later realized we have a comment with an additional action for the upload file as well. Need to come up with a better design so we can create a separate ToolbarJsonEditor component with support for custom actions.

Reply:
Since the actions for each button will need to be passed in anyway, you could have the JSON component take "children" that get yielded within the toolbar. Or, if we know we want a discreet number of options (probably not) we could to a contextual component. Passing in the yielded content would be simpler, but it does depend on the developer using the correct class names for consistency in styling

Reply:
Done. Moved the Toolbar to json-editor component itself and added a yield block for custom actions.